server/src/test/java/org/elasticsearch/action/fieldcaps/FieldCapabilitiesRequestTests.java

```diff
@@ -93,4 +95,11 @@ public void testFieldCapsRequestSerialization() throws IOException {
             assertEquals(deserialized.hashCode(), request.hashCode());
         }
```

Comment:
maybe while at it we could make this test extend `AbstractWireSerializingTestCase`, it would save us some code and make things more consistent. Could also be a followup though.

Reply:
Makes sense, will also do in the follow-up PR.

Reply:
Addressed in https://github.com/elastic/elasticsearch/pull/30181 (converting the test actually caught a bug).